diff -ur pstoedit-3.70/configure.ac pstoedit-3.70-im7/configure.ac
--- pstoedit-3.70/configure.ac	2014-12-28 21:39:18.000000000 +0100
+++ pstoedit-3.70-im7/configure.ac	2016-12-25 22:16:29.147826432 +0100
@@ -299,7 +299,7 @@
 else
       if test -n "$PKG_CONFIG"; then
         AC_MSG_CHECKING(ImageMagick flags through pkg-config)
-	PKG_CHECK_MODULES(LIBMAGICK, ImageMagick++, 
+	PKG_CHECK_MODULES(LIBMAGICK, Magick++, 
 		HAVE_LIBMAGICK=yes 
 		LIBMAGICK_CFLAGS="-DHAVE_MAGIC $LIBMAGICK_CFLAGS" 
 		LIBMAGICK_LDFLAGS="$LIBMAGICK_LIBS" , 
