--- pstoedit-3.78/src/pstoedit.cpp.orig	2022-04-11 17:09:22.797484174 +0200
+++ pstoedit-3.78/src/pstoedit.cpp	2022-04-11 17:09:25.547439114 +0200
@@ -267,30 +267,6 @@ static void loadpstoeditplugins(const ch
 		//pluginsloaded = true;
 	}
 
-	// If the above failed, also look in the directory where the pstoedit .exe/dll was found
-	if (!pluginsloaded) {
-	  char szExePath[1000];
-	  szExePath[0] = '\0';
-	  const unsigned long r = P_GetPathToMyself(progname, szExePath, sizeof(szExePath));
-	  if (verbose)  errstream << "pstoedit : path to myself:" << progname << " " << r << " " << szExePath<< endl;
-	  char *p = nullptr;
-	  if (r && (p = strrchr(szExePath, directoryDelimiter)) != nullptr) {
-		*p = '\0';
-		pluginsloaded |= loadPlugInDrivers(szExePath, errstream,verbose);
-	  }
-	  // now try also $exepath/../lib/pstoedit
-      if (szExePath[0]) {
-         // it is not an empty string
-#if COMPILEDFOR64BIT 
-	    strcat_s(szExePath,1000,"/../lib64/pstoedit");
-#else
-	    strcat_s(szExePath,1000,"/../lib/pstoedit");
-#endif
-	    if (!strequal(szExePath, plugindir.c_str())) {
-	      pluginsloaded |= loadPlugInDrivers(szExePath, errstream,verbose);
-	    }
-	  }
-	}
 #ifdef PSTOEDITLIBDIR
 	if (!pluginsloaded) {
 		struct stat s;
