--- pstoedit-4.01/src/pstoedit.cpp.orig	2024-04-24 22:14:51.433955369 +0200
+++ pstoedit-4.01/src/pstoedit.cpp	2024-04-24 22:16:13.650176633 +0200
@@ -283,29 +283,6 @@ void loadpstoeditplugins(const char *pro
 		//pluginsloaded = true;
 	}
 
-	// If the above failed, also look in the directory where the pstoedit .exe/dll was found
-	if (!pluginsloaded) {
-	  char szExePath[1000] = { 0 };
-	  const unsigned long r = P_GetPathToMyself(progname, szExePath, sizeof(szExePath));
-	  if (verbose)  errstream << "pstoedit : path to myself:" << progname << " " << r << " " << szExePath<< endl;
-	  char *p = nullptr;
-	  if (r && (p = strrchr(szExePath, directoryDelimiter)) != nullptr) {
-		*p = '\0';
-		pluginsloaded |= loadPlugInDrivers(szExePath, errstream,verbose);
-	  }
-	  // now try also $exepath/../lib/pstoedit
-      if (szExePath[0]) {
-         // it is not an empty string
-#if COMPILEDFOR64BIT 
-	    strcat_s(szExePath,1000,"/../lib64/pstoedit");
-#else
-	    strcat_s(szExePath,1000,"/../lib/pstoedit");
-#endif
-	    if (!strequal(szExePath, plugindir.c_str())) {
-	      pluginsloaded |= loadPlugInDrivers(szExePath, errstream,verbose);
-	    }
-	  }
-	}
 #ifdef PSTOEDITLIBDIR
 	if (!pluginsloaded) {
 		struct stat s;
