--- pstoedit-3.40/configure.ac.orig	2005-01-07 20:14:33.716016000 +0100
+++ pstoedit-3.40/configure.ac	2005-01-07 20:33:31.743010240 +0100
@@ -352,7 +352,7 @@
     AC_SUBST(LDLIBS)
   else
     AC_MSG_RESULT([GNU])
-    SPECIAL_CXXFLAGS="-DBUGGYGPP -pedantic -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith "
+    SPECIAL_CXXFLAGS="-DBUGGYGPP -Wall -Wwrite-strings -Wcast-qual -Wpointer-arith "
   fi
 elif test `echo "$CXX" | egrep -c 'xlC$'` -eq 1; then
   AC_MSG_RESULT([AIX xlC])
