--- pstoedit-3.40/src/Makefile.am.orig	2004-11-03 22:05:16.000000000 +0100
+++ pstoedit-3.40/src/Makefile.am	2005-01-07 20:14:00.275100432 +0100
@@ -63,27 +63,27 @@
 	drvtk.cpp	drvtk.h                 \
 	$(specific_drivers_src)                 \
         $(sample_drivers_src)                   
-libp2edrvstd_la_LIBADD=-L. libpstoedit.la
+libp2edrvstd_la_LIBADD=libpstoedit.la
 libp2edrvstd_la_LDFLAGS=-no-undefined -module
 
 libp2edrvlplot_la_SOURCES=drvlplot.cpp drvlplot.h initlibrary.cpp
-libp2edrvlplot_la_LIBADD=$(LIBPLOTTER_LDFLAGS) -L. libpstoedit.la
+libp2edrvlplot_la_LIBADD=$(LIBPLOTTER_LDFLAGS) libpstoedit.la
 libp2edrvlplot_la_LDFLAGS=-no-undefined -module
 
 libp2edrvswf_la_SOURCES=drvswf.cpp drvswf.h initlibrary.cpp
-libp2edrvswf_la_LIBADD=$(LIBMING_LDFLAGS) -L. libpstoedit.la
+libp2edrvswf_la_LIBADD=$(LIBMING_LDFLAGS) libpstoedit.la
 libp2edrvswf_la_LDFLAGS=-no-undefined -module
 
 libp2edrvmagick___la_SOURCES=drvmagick++.cpp drvmagick++.h initlibrary.cpp
-libp2edrvmagick___la_LIBADD=-L. libpstoedit.la
+libp2edrvmagick___la_LIBADD=libpstoedit.la
 libp2edrvmagick___la_LDFLAGS=-no-undefined ${LIBMAGICK_LDFLAGS}  -module
 
 if USE_EMFBYSOURCE
 libp2edrvwmf_la_SOURCES=emflib.cpp drvwmf.cpp drvwmf.h initlibrary.cpp
-libp2edrvwmf_la_LDFLAGS=-no-undefined -L. libpstoedit.la -module
+libp2edrvwmf_la_LDFLAGS=-no-undefined libpstoedit.la -module
 else
 libp2edrvwmf_la_SOURCES=drvwmf.cpp drvwmf.h initlibrary.cpp
-libp2edrvwmf_la_LDFLAGS=-no-undefined $(LIBEMF_LDFLAGS) -L. libpstoedit.la -module
+libp2edrvwmf_la_LDFLAGS=-no-undefined $(LIBEMF_LDFLAGS) libpstoedit.la -module
 endif
 
 libpstoedit_la_SOURCES =			\
@@ -101,11 +101,11 @@
 	psfront.h				\
 	version.h
 
-libpstoedit_la_LIBADD = ${CXX_STD_LIB} ${CXX_RUNTIME_LIB}
+libpstoedit_la_LIBADD = ${CXX_STD_LIB} ${CXX_RUNTIME_LIB} ${LIBLD_LDFLAGS}
 libpstoedit_la_LDFLAGS = -no-undefined
 
 pstoedit_SOURCES = cmdmain.cpp
-pstoedit_LDFLAGS = -no-undefined -L. libpstoedit.la ${LIBLD_LDFLAGS}
+pstoedit_LDFLAGS = -no-undefined libpstoedit.la ${LIBLD_LDFLAGS}
 
 pkginclude_HEADERS =				\
 	pstoedit.h				\
